.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-max-line-length = 160
-exclude = sandbox
+max-line-length = 99
```

Comment:
why not 25? I am curious why 99 was chosen, but it is a bit arbitrary.

Reply:
I was already at 100 and @facundobatista pointed out that [pep8] says,
> Some teams strongly prefer a longer line length. For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the line length limit up to 99 characters, provided that comments and docstrings are still wrapped at 72 characters.

I think we're still not doing the second half of that but we might want to aim for it at some point (not here though, this is already big enough a diff).

[pep8]: https://www.python.org/dev/peps/pep-0008/#id19